pkg-diff.sh

```diff
@@ -572,7 +572,11 @@ check_single_file()
           diff -u old/flist new/flist
           return 1
        fi
-       flist=`grep date new/flist | sed -e 's,.* date ,,'`
+       if [ "$UNJAR" = unzip ] ; then
```

Comment:
still buggy - the variable is unset because `unjar_l` is called in a subprocess (because of the `| sort`)